spec/controllers/state_file/questions/w2_controller_spec.rb

```diff
@@ -198,4 +198,32 @@
       end
     end
   end
-end
\ No newline at end of file
+
+  describe "#load_w2" do
+    context "when box14_ui_wf_swf is nil and box14_ui_hc_wd is not" do
+      let(:intake) { create :state_file_nj_intake }
+      let!(:state_file_w2) do
+        create :state_file_w2, state_file_intake: intake, w2_index: 1,
+               box14_ui_wf_swf: nil, box14_ui_hc_wd: 340
+      end
+  
+      it "sets box14_ui_wf_swf to box14_ui_hc_wd" do
+        get :edit, params: { id: state_file_w2.id }
+        expect(assigns(:w2).box14_ui_wf_swf).to eq 340
```

Comment:
that's a good point -- the value of `box14_ui_hc_wd` doesn't end up mattering because it is just there as a backup in case `box14_ui_wf_swf` is empty, then isn't used anymore. but i think I still want an expectation of it's value for the same reason as you. so i'll consider setting it to `nil` after it's used and then put that in the expectation...

Reply:
That's a good point-- `box14_ui_hc_wd` doesn't matter from here on out so I didn't bother adding an expectation for it. I will be making an update to the w2 model that will help make it clear that we aren't using that value...

Reply:
to resolve this, i introduced a function called `get_box14_ui_overwrite()` in `state_file_w2` so this logic is not in this controller anymore